atomix/cluster/src/test/java/io/atomix/cluster/messaging/impl/NettyUnicastServiceTest.java

```diff
@@ -55,6 +56,18 @@ public void testUnicast() throws Exception {
     await(5000);
   }
 
+  @Test
+  public void shouldNotThrowExceptionWhenServiceStopped() throws Exception {
+    // given
+    service2.stop();
+
+    // when
+    service2.unicast(address1, "test", "Hello world!".getBytes());
+
+    // then
+    assertTrue(true);
```

Comment:
Maybe better:

`Assertions.assertThatCode(() -> toTest.method()).doesNotThrowAnyException();` or 

```
@Test(expected = Test.None.class /* no exception expected */)
public void test_printLine() {
    Printer.printLine("line");
}
```

Reply:
Yeah I searched yesterday evening for that but was not able to find it idk maybe was to late. Thanks